### PR TITLE
Icons in sidebar current view

### PR DESCRIPTION
**Pull request by HamutalCohen3**
I used the icons the same way they were used in the map, selecting them through their title. Though it seems like there are icons that don't get to appear, such as pedestrian.

### CODE REVIEW

File: static/js/sidebar.js [hunk 1]
```diff
@@ -43,14 +43,11 @@ var SidebarView = Backbone.View.extend({
                 }
 
                 var iconUrl = markerView.getIcon();
-                if (isRetina){
```
Comment (danielhers) [inline static/js/sidebar.js]: I think this is required for #180. Why did you remove it?

**Comment by danielhers** [comment]
See @omerxx 's comment from September: https://github.com/hasadna/anyway/issues/444#issuecomment-142292436
What is the tradeoff between your option and his?

**Comment by HamutalCohen3** [comment]
About the isretina I'm not sure.
But I thought that its best to get the icons the same way we get it on the
map so there wouldn't be any differences in the sidebar. The function omer
suggested gets the icon through its severity and type attributes and in the
map the icons get the style from the title attribute and css selectors.

**Comment by danielhers** [comment]
Right, so I think that the mechanism to get the icon by severity/type is obsolete and should be removed.
Either that, or the CSS mechanism to get the icon by the title should be replaced with a more robust one (that also supports translation to other languages). @LironRS , for you consideration.
Anyway, this will be a different issue. I'll merge this one now.
